IndomitableIbises/Bangazon-Sprint-3
# Faker ea

## Link to Ticket
Closes # [Link](https://github.com/IndomitableIbises/Bangazon-Sprint-3/issues/27)

## Description of Proposed Changes

- This ticket creates the faker data for team usage

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout faker-ea
```

```
1. on your computer, go to usr/local/bin
2. add a new file "django_data.sh
3.  place the script from "[Item #2: The shell script](https://github.com/nashville-software-school/bangazon-llc/blob/master/DB_RESET_SEED_SYSTEM.md)" into the new file.
4.  pip install django_seed
5.  run 'django_data.sh website faker_factory.py' from the terminal
6.  open dbbrowser and verify there is data for models (Note:  may need to uncomment out models you don't have)
```





## Impacted Areas in Application

List general components of the application that this PR will affect:

* The faker_factory.py file will affect all tables within the database with the exception of shopping cart.  After merging this file, you should see seeded data in the database.

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
